cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/Postgres.kt

```diff
@@ -33,6 +33,7 @@ class PostgresDataSourceFactory(
     jdbcUrl = "jdbc:postgresql://${pgHost}:${pgPort}/dev_all_projects"
   }
   private val superDataSource = HikariDataSource(superConfig)
+  private val cachedDataSources: MutableMap<ProjectRefid, DataSource> = mutableMapOf()
```

Comment:
I don't think it makes too much sense now because we're using the same database for all projects anyway. It will make sense later, if and when we start using different databases. But as soon as there is just one database, one data source should be enough

Reply:
Do you mean changing the schema every time the database is accessed?

Reply:
We will do a lot of schema tricks in the normal mode anyway, even if we have one database per project. When there are many projects in the same database, we can keep their actual versions in different schemas too.